homeassistant/components/netatmo/data_handler.py

```diff
@@ -60,7 +59,7 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry):
         self.listeners: list[CALLBACK_TYPE] = []
         self._data_classes: dict = {}
         self.data = {}
-        self._queue: Deque = deque()
+        self._queue: deque = deque()
```

Comment:
Is this typing still needed? Doesn't mypy understand the type when assigning the instance to the variable? We don't provide any detailed type information for the contents of the deque.

Reply:
Yeah, I think so. Good catch